app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -6,6 +6,20 @@ module HostManagedExtensions
       include ForemanTasks::Concerns::ActionSubject
 
       module Overrides
+        def update(attrs)
+          if attrs[:content_facet_attributes]
+            cv_id = attrs[:content_facet_attributes].delete(:content_view_id)
+            lce_id = attrs[:content_facet_attributes].delete(:lifecycle_environment_id)
+            if cv_id && lce_id
+              content_facet.assign_single_environment(content_view_id: cv_id, lifecycle_environment_id: lce_id)
+            end
+            if (cv_id.present? && lce_id.blank?) || (cv_id.blank? && lce_id.present?)
+              fail "content_view_id and lifecycle_environment_id must be provided together"
```

Comment:
Although the patch seems to fix the issue, I've got a semi-related question:

If I've created a host within a hostgroup which has lifecycle env and content view attached, can I remove prefiled and saved these inherited environment and content view from the host?

Reply:
Theoretically you should be able to click the X and remove the value. That whole area is buggy right now though; also @lfu is currently working on https://projects.theforeman.org/issues/35834 which will refactor that whole area of the form.